assets/js/translator.js

```diff
@@ -446,22 +449,26 @@ function initChainGraph() {
         .attr('fill', '#999');
 
     simulation = d3.forceSimulation()
-        .force('link', d3.forceLink().id(function (d) { return d.id; }).distance(nodeSize * 10))
+        .force('link', d3.forceLink().id(function (d) { return d.id; }).distance(CHAIN_NODE_SIZE * 10))
         .force('charge', d3.forceManyBody().strength(-700))
-        .force('center', d3.forceCenter(width / 2, height / 2))
+        .force('center', d3.forceCenter(CHAIN_CHOOSER_WIDTH / 2, CHAIN_CHOOSER_HEIGHT / 2))
         .alphaDecay(0.018);
     /* eslint-enable no-magic-numbers */
 
-    refreshChainGraph();
+    refreshChainGraphAndChosenPath();
 }
 
 function boundary(dist, max) {
-    if(dist < nodeSize) return nodeSize;
-    if(dist < max - nodeSize) return dist;
-    return max - nodeSize;
+    if(dist < CHAIN_NODE_SIZE) return CHAIN_NODE_SIZE;
+    if(dist < max - CHAIN_NODE_SIZE) return dist;
+    return max - CHAIN_NODE_SIZE;
 }
 
 function getAllPaths(src, trgt, curPath, seens) {
```

Comment:
All the `trgt`s (possibly even outside my changes) should be changed into `dst`s, right? I'll leave this as a final change after all everything else is settled because the more important changes will be swamped out.

Reply:
> All the trgts (possibly even outside my changes) should be changed into dsts, right?

Yep.

> I'll leave this as a final change after all everything else is settled because the more important changes will be swamped out.

Up to you! I review the entire diff again either way.

Reply:
I realized APy returns objects with the "target" terminology, not "destination". It makes more sense to change html-tools to match APy, right?

Reply:
We could but I'm somewhat opposed to this because it involves changing a lot of existing code that is relatively brittle, especially without tests.